experimental/PlaneCurve/AffinePlaneCurve.jl

```diff
@@ -109,7 +180,7 @@ function curve_intersect(C::AffinePlaneCurve{S}, D::AffinePlaneCurve{S}) where S
         push!(Y, -f + gen(R, 2))
      end
   end
-     # For each y, we compute the possible values of x by replacing the second
+     # For each y, we compute the possible values of x by jldoctestacing the second
```

Comment:
Search and replace gone wrong? :-)
```suggestion
     # For each y, we compute the possible values of x by replacing the second
```

Reply:
Indeed ;^^